x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -87,6 +88,16 @@ export const getFormattedEntries = (entries: EntriesArray): FormattedEntry[] =>
   return formattedEntries.flat();
 };
 
+export const getEntryValue = (entry: Entry): string | string[] | null => {
+  if (entriesList.is(entry)) {
+    return entry.list.id;
```

Comment:
NOTE: This is temporary, didn't want to dig into changing UI code in this PR that's already large. Thought maybe we wanted to add `name` to the the `EntryList` for ease of use here, but that would also then require any updates to large lists to search for references to it in exception list items.